src/main/java/ml/shifu/shifu/container/obj/ModelConfig.java

```diff
@@ -278,6 +278,8 @@ public static ModelConfig createInitModelConfig(String modelName, ALGORITHM alg,
             trainConf.setNumTrainEpochs(20000);
         } else if(ALGORITHM.LR.equals(alg)) {
             trainConf.setNumTrainEpochs(100);
+        } else if(ALGORITHM.TF_DNN.equals(alg)) {
```

Comment:
If it could be changed to TF, in the future i would like to scale to RNN LSTM...